extensions/sql-database-projects/src/tools/buildHelper.ts

```diff
@@ -139,7 +139,7 @@ export class BuildHelper {
 		if (sqlProjStyle === ProjectType.SdkStyle) {
 			return ` build ${projectPath} /p:NetCoreBuild=true /p:SystemDacpacsLocation=${buildDirPath}`;
 		} else {
-			return ` build ${projectPath} /p:NetCoreBuild=true /p:NETCoreTargetsPath=${buildDirPath}`;
+			return ` build ${projectPath} /p:NetCoreBuild=true /p:NETCoreTargetsPath=${buildDirPath} /p:SystemDacpacsLocation=${buildDirPath}`;
```

Comment:
@kisantia 

> The old project XML logic in ADS used to add system db references for legacy projects with the variable `$(NETCoreTargetsPath)`, so the path would look like` $(NETCoreTargetsPath)\SystemDacpacs\160\master.dacpac`. The new projects library adds system db references for both legacy and SDK-style projects with the variable `$(SystemDacpacsLocation)`, so now this variable also needs to get passed in when building legacy projects.

Is this a good change in DacFx, or should we (instead) bring the behavior back to how it was before?  I clearly missed this difference, and am wondering about the benefits to fixing this at the DacFx level vs. the ADS level.

Reply:
I think good change - the name `SystemDacpacsLocation` makes more sense if people are looking at their .sqlproj and at least the path is now the same for SDK-style and legacy projects. The changes before this were preview, so it should be ok to change this, especially since older projects that use `$(NETCoreTargetsPath)` will still build in ADS after this change.

Reply:
Sounds good to me!